platforms/sagemaker-studio/README.md

```diff
@@ -0,0 +1,104 @@
+After configuring `ecr_configure.sh` variables as described in previous section, you will be able to create a Docker image using the available Dockerfile file, and this script will build it and upload it to your ECR. Just run:
+
+`./ecr_configure.sh`
+
+If it does not have execution rights, just use `bash ./ecr_configure.sh` or grant permissions by executing `chmod +x ecr_configure.sh`
+
+Don't close the terminal, you will need the output (especially the **image URI**) for adding it to Sagemaker.
+
+After running `.sh` file, please make sure these **outputs are present:**
+
+- `Login Succeeded `
+
+- ` Image successfully tagged`
+
+-  `Pushed the image properly`
+
+- `ImageVersionStatus`  is `CREATED` :
+ ```
+{
+    "BaseImage": "ACCOUNT_ID.dkr.ecr.REGION.amazonaws.com/jsl-ocr:sparknlp-image",
+    "ContainerImage": "ACCOUNT_ID.dkr.ecr.REGION.amazonaws.com/jsl-ocr@sha256:eXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "CreationTime": "2021-12-15T13:24:31.162000+00:00",
+    "ImageArn": "arn:aws:sagemaker:REGION:ACCOUNT_ID:image/sparknlp-image",
+    "ImageVersionArn": "arn:aws:sagemaker:REGION:ACCOUNT_ID:image-version/sparknlp-image/1",
+    "ImageVersionStatus": "CREATED",
+    "LastModifiedTime": "2021-12-15T13:24:31.525000+00:00",
+    "Version": 1
+}
+```
+
+**NOTE:** If the REPO_NAME is an existing name in your ECR, first command will show a warning that it already exists. You can ignore it.
+
+## 5. Open Sagemaker
+In Sagemaker, go to `SageMaker Domain` -> `Studio`. Make sure you have an active user, with the role arn you have set in ROLE_ARN in Section 3. 
+
+On the bottom part of the screen, you will see `Customer SageMaker Studio images attached to domain`.
+
+## 6. Attach image
+In the `Choose image source`, click on `New Image`, and then in the `Enter an ECR image URI`, add the `URI` you will see in the logs from step 4. Click on "Next".
+
+Then, in `Image properties`, fill `Image name` (same as IMAGE_NAME in step 3), and `Image display name` (also use IMAGE_NAME). In `IAM role`, make sure you select the same `ROLE_ARN` from section 3 in the dropdown. Click on "Next".
+
+`EFS mount path` should be `/root`. Kernel name should be `conda-env-myenv-py`. Kernel display name should be `Python [conda env: myenv]`. Click on "Advanced Configuration". Set `User ID (UID)` to 0. Set `Group ID (GUID)` to 0. Click on Submit.
+
+## 7. Launching Sagemaker Studio
+After clicking on submit, you  will see a **Ready** status in the `Sagemaker Domain -> Studio` screen. Go to Users (top of the screen), click on `Launch app` and select `Studio`.
+
+Sagemaker Studio will take some time to launch, because will instantiate a container of the image we have added. After that, you can start a new notebook. Make sure you see as active the `Python [conda env: myenv]` kernel. It may take also some time to spin.
+
+### 7.1 Important Note:
+**IMPORTANT**
+**Please make sure very first command in the notebook should be:**
```

Comment:
I think `Please make sure the very first command in the notebook is:` sounds better